.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -147,7 +149,7 @@ jobs:
         # So we simply expect the developer and reviewers to look over the test report, deployed
         # below, before merging.
         continue-on-error: true
-        run: pnpm exec playwright test --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
+        run: pnpm exec playwright test --only-changed origin/main --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
```

Comment:
We'll see how it goes and do a comparison when we have all the test suites in place. But we may not need sharding if we use `--only-changed`.

Reply:
Oh, nice find!